src/main/java/core/basesyntax/RandomFigure.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomFigure {
+    private static final int MAX_FIGURE_NUMBER = 5;
+    private final Random random = new Random();
+    private Figure figure;
+
+    public Figure getRandomFigures() {
+
+        switch (random.nextInt(MAX_FIGURE_NUMBER)) {
+            case 0:
+                figure = new Circle();
+                break;
+            case 1:
+                figure = new IsoscelesTrapezoid();
+                break;
+            case 2:
+                figure = new Rectangle();
+                break;
+            case 3:
+                figure = new RightTriangle();
+                break;
+            case 4:
+                figure = new Square();
+                break;
+            default:
+                break;
+        }
+        return figure.getRandomFigure();
```

Comment:
return switch like this
```suggestion
        return switch (random.nextInt(MAX_FIGURE_NUMBER)) {
            case 0 -> new Circle();
            case 1 -> new IsoscelesTrapezoid();
            case 2 -> new Rectangle();
            case 3 -> new RightTriangle();
            default -> new Square();
        };
```
remove this class, move logic to figure supplier, no need to have two classes for that
when you set up those figures, pass random properties, random color